packages/react-dom/src/__tests__/ReactCompositeComponent-test.js

```diff
@@ -63,56 +65,154 @@ describe('ReactCompositeComponent', () => {
   beforeEach(() => {
     jest.resetModules();
     React = require('react');
-    ReactDOM = require('react-dom');
+    ReactDOMX = require('react-dom');
+    ReactDOMClient = require('react-dom/client');
     ReactCurrentOwner =
       require('react').__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED
         .ReactCurrentOwner;
-    ReactTestUtils = require('react-dom/test-utils');
-    PropTypes = require('prop-types');
+    Scheduler = require('scheduler');
+    assertLog = require('internal-test-utils').assertLog;
+    act = require('internal-test-utils').act;
+  });
+
+  describe('MorphingComponent', () => {
+    let instance;
+    let childInstance;
 
-    MorphingComponent = class extends React.Component {
-      state = {activated: false};
+    beforeEach(() => {
+      MorphingComponent = class extends React.Component {
```

Comment:
These helper components are only used in a few tests, so I moved them to their own `describe`block so they're not re-created every run.